cmd/ci-operator-prowgen/main.go

```diff
@@ -22,6 +22,7 @@ import (
 
 const (
 	prowJobLabelVariant = "ci-operator.openshift.io/variant"
+	sentryDsnName       = "sentry-dsn"
```

Comment:
Can we call this out as a volume name?

Reply:
Yeah. It served also as a secret name, so I made a separate constant for it.